Dockerfile

```diff
@@ -20,24 +27,26 @@ RUN go mod download
 ADD . .
 
 ARG opts
-RUN env ${opts} make build
+RUN env ${opts} make build-static
+RUN setcap 'cap_net_bind_service=+ep' /src/bin/blocky
+
+RUN adduser -S -D -H -u 10001 -h /app -s /sbin/nologin blocky
 
 # final stage
-FROM alpine:3.16
+FROM scratch
 
 LABEL org.opencontainers.image.source="https://github.com/0xERR0R/blocky" \
       org.opencontainers.image.url="https://github.com/0xERR0R/blocky" \
       org.opencontainers.image.title="DNS proxy as ad-blocker for local network"
 
 COPY --from=build-env /src/bin/blocky /app/blocky
-RUN apk add --no-cache ca-certificates bind-tools tini tzdata libcap && \
-    adduser -S -D -H -h /app -s /sbin/nologin blocky && \
-    setcap 'cap_net_bind_service=+ep' /app/blocky
-
-HEALTHCHECK --interval=1m --timeout=3s CMD dig @127.0.0.1 -p 53 healthcheck.blocky +tcp +short || exit 1
+COPY --from=build-env /etc/passwd /etc/passwd
 
 USER blocky
 WORKDIR /app
 
-ENTRYPOINT ["/sbin/tini", "--"]
-CMD ["sh", "-c", "/app/blocky --config ${CONFIG_FILE:-/app/config.yml}"]
+ENV BLOCKY_CONFIG_FILE=/app/config.yml
```

Comment:
CONFIG_FILE was changed to BLOCKY_CONFIG_FILE, it could potentially break existing installations

Reply:
As the environment will be directly read through blocky it is good practice to prefix environment variables.
There is a fix for backward compatibility in root.go.

Reply:
Added fallback to old var if new var isen't used in [initconfig](https://github.com/kwitsch/blocky-dev/blob/fb-dockeroptimization/cmd/root.go#L69).